iotedgedev/dotnetmoduleprocessor.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+
+class DotNetModuleProcessor(object):
+    def __init__(self, envvars, utility, output):
+        self.envvars = envvars
+        self.utility = utility
+        self.output = output
+
+    def build(self, module_dir):
+        project_files = [os.path.join(module_dir, f) for f in os.listdir(

```

Comment:
This code is used in both build and publish.  Please refactor into a method that returns the first project file given module_dir.